app/views/components/check_uk_visa/_result-card.html.erb

```diff
@@ -0,0 +1,60 @@
+<%
```

Comment:
Is this component intended to be reused? If not, then maybe the best place is: https://github.com/alphagov/smart-answers/tree/main/app/flows/check_uk_visa_flow, as app/components feels like a shared directory.

Reply:
I believe this component is intended to be reused in future work, although all within the check-uk-visa flow. I think there are plans for a mix of generic and flow-specific components, so `app/views/components` is being used to hold all these templates and the `app/flows/*` folders will hold "content" templates.

Reply:
Ah okay, thanks for explaining!